docs/security/seccomp.md

```diff
@@ -3,27 +3,25 @@
 title = "Seccomp security profiles for Docker"
 description = "Enabling seccomp in Docker"
 keywords = ["seccomp, security, docker, documentation"]
+[menu.main]
+parent= "smn_secure_docker"
 +++
 <![end-metadata]-->
 
-Seccomp security profiles for Docker
-------------------------------------
+# Seccomp security profiles for Docker
 
-The seccomp() system call operates on the Secure Computing (seccomp)
-state of the calling process.
+Secure computing mode (Seccomp) is a Linux kernel feature. You can use it to
+restrict the actions available within the container.  The `seccomp()`
+system call operates on the seccomp state of the calling process. You can use this feature to restrict the access of your application.  
 
-This operation is available only if the kernel is configured
-with `CONFIG_SECCOMP` enabled.
+This feature is available only if the kernel is configured
+with `CONFIG_SECCOMP` enabled.  
 
-This allows for allowing or denying of certain syscalls in a container.
+## Passing a profile for a container
 
-Passing a profile for a container
----------------------------------
-
-Users may pass a seccomp profile using the `security-opt` option
-(per-container).
-
-The profile has layout in the following form:
+The default seccomp profile provides a sane default for running containers with
+seccomp. It is moderately protective while providing wide application
+compatibility. The default Docker profile has layout in the following form:
```

Comment:
So it's unrelated :)


On Jan 3, 2016, 15:47 -0800, Sebastiaan van Stijn<notifications@github.com>, wrote:
>  
> Indocs/security/seccomp.md(https://github.com/docker/docker/pull/18452#discussion_r48698986):
>  
> >>-Passing a profile for a container>---------------------------------->->-Users may pass a seccomp profile using the `security-opt` option>-(per-container).>->-The profile has layout in the following form:>+The default seccomp profile provides a sane default for running containers with>+seccomp. It is moderately protective while providing wide application>+compatibility. The default Docker profile has layout in the following form:
>  
> @jfrazelle(https://github.com/jfrazelle)does the example format below need to be updated if we switch to a whitelist? (#18979(https://github.com/docker/docker/pull/18979)), I.e. will"defaultAction": "SCMP_ACT_ALLOW"be a "deny"?
>  
>  
> —
> Reply to this email directly orview it on GitHub(https://github.com/docker/docker/pull/18452/files#r48698986).
>  
>  
>